### PR TITLE
Population dataset parse neaf 36

### PR DESCRIPTION
**Pull request by VladimirBaklan**

### CODE REVIEW

File: src/main/scala/com/itechart/ny_accidents/service/DistrictsService.scala [hunk 1]
```diff
@@ -31,4 +31,16 @@ class DistrictsService {
...
+
+  def getDistrict(districtName: String, districts: Seq[District]): Option[District] = {
+    districts.find(_.districtName.equalsIgnoreCase(districtName)) match {
+      case Some(value) => Some(value)
+      case None =>
+        districts.find(dist =>
+        StringUtils.getLineMatchPercentage(
+          dist.districtName.toLowerCase.replaceAll(" ", "_"),
+          districtName.toLowerCase().replaceAll(" ", "_")
+        ) > 90.0)
```
Comment (sleshJdev) [inline src/main/scala/com/itechart/ny_accidents/service/DistrictsService.scala]: 90.0 it's kind of "magic number". Could you please create a constant for this literal value and give it meaningful name?

File: src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala [hunk 2]
```diff
@@ -11,4 +12,16 @@ object StringUtils {
     }
   }
 
+  def getLineMatchPercentage(firstString: String, secondString: String): Double = {
```
Comment (sleshJdev) [inline src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala:15]: I'd like to see a couple of unit test for this tricky method. Could you please add a few? Thanks

**Review by sleshJdev — CHANGES_REQUESTED** [review]

File: src/main/scala/com/itechart/ny_accidents/service/DistrictsService.scala (hunk 1 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/service/DistrictsService.scala]: Fixed

File: src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala (hunk 2 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala:15]: I'm not sure about usability of this method, but tests were written

**Review by sleshJdev — APPROVED** [review]

**Review by AlexKostritsa — APPROVED** [review]

**Pull request merged**